### PR TITLE
20 feature login basiuk

### PR DESCRIPTION
**Pull request by DMTRIP**

### CODE REVIEW

**Review by Viktor-Hordynskyi — APPROVED** [review]

**Review by victorpoliushko — APPROVED** [review]
Advices:
- format your src/components/page/login/login.js file
- add " text-align: center; " for class title in src/components/page/login/login.scss

File: src/components/header/header.js [hunk 1]
```diff
@@ -5,8 +5,10 @@ import logo from './../../img/logo.svg';
 import search from './../../img/search.svg';
 import Guest from './guest';
 import User from './user';
+import isAuth from "../hocs/is-auth";
 
-const Header = () => {
+const Header = ({ isAuth }) => {
+  console.log(isAuth);
```
Comment (AndriiDruzhytskyi) [inline src/components/header/header.js]: Do you need this console output in dev?

File: src/components/page/login/login.js [hunk 2]
```diff
@@ -42,67 +48,34 @@ const Login = () => {
...
+                <div className="login">
+                    <h1 className='title'>Login</h1>
                             <Formik
                                 initialValues={{
                                     email: '',
                                     password: ''
                                 }}
                                 validationSchema={LoginSchema}
                                 onSubmit={submit}
+                                onFocu
```
Comment (AndriiDruzhytskyi) [inline src/components/page/login/login.js]: Is this a typo??

**Review by AndriiDruzhytskyi — CHANGES_REQUESTED** [review]
I have some problems when try to pass request to server, please tell me more about your configuration via Slack.
image

File: src/components/header/header.js (hunk 1 above)
Comment (DMTRIP) [reply src/components/header/header.js]: No I'll remove the line

File: src/components/page/login/login.js (hunk 2 above)
Comment (DMTRIP) [reply src/components/page/login/login.js]: +

**Review by AndriiDruzhytskyi — APPROVED** [review]
After this pull request merged:

1. We must resolve port number mismatch on anouther task.
2. Resolve routing when user set tokens locally.
3. Resolve routing when triggers a refresh token logic.

**Pull request merged**